src/components/Link.react.js

```diff
@@ -81,6 +87,7 @@ export default class Link extends Component {
                 href={href}
                 onClick={e => this.updateLocation(e)}
                 title={title}
+                target={external ? '_blank' : '_self'}
```

Comment:
I think that I'd prefer that we just stick to the HTML convention here instead of defining a new API. That way, this component is more or less 1-1 with `html.A`. So, just let users pass through `target` directly rather than defining a new `external` property.

Reply:
I've seen a couple of other link components define external as a separate property, but I think you raise a good point, for both maintaining parity with `html.A` and offering a bit more freedom for the user to decide what to target. 

Changed in 6514c83.

Reply:
> I think that I'd prefer that we just stick to the HTML convention here

I agree @chriddyp - cc @Marc-Andre-Rivet - we had a similar discussion a couple of days ago, also because there *are* other values of `target` that still matter.